### PR TITLE
5 view review

### PR DESCRIPTION
**Pull request by RyanNolan1**

### CODE REVIEW

File: src/components/SingleReview.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    return (
+      <div className="review-container">
+         <img
+                src={review.review_img_url}
+                alt={review.title}
+                className="single-review-image"
+              />
+        <h2 className="review-header"> {review.title} </h2>
+        <h3 className="review-category"> {review.category}</h3>
+        <p className="review-body"> {review.review_body} </p>
```
Comment (Slyose) [inline src/components/SingleReview.jsx:25]: Good work so far, but missing a few properties. The articles should also show when they were created_at, the votes, and the author. Refer to the ticket for a list of properties that should be included.

**Review by Slyose — COMMENTED** [review]
Needs a few more properties to be rendered

File: src/api.js [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
   return axios
     .get("https://eager-getup-ant.cyclic.app/api/reviews")
     .then(({ data: { reviews } }) => {
       return reviews;
     });
 };
+
+export const getSingleReview = ( review_id ) => {
+  return gameApi.get(`reviews/${review_id}`).then((res) => {
+      console.log(res.data)
```
Comment (yangalexyangg) [inline src/api.js:18]: remove log please

**Review by yangalexyangg — COMMENTED** [review]
LOoks really good to me  happy to approve : )

**Pull request merged**